data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+			`	From the shaking voice, you can tell he's frightened.`
+			`	"C-can I help you?"`
+			choice
+				`	"You're a Hai. Why are you this far into human space?"`
+					goto far
+				`	"Do you need a ride home to Hai space?"`
+					goto "ride offer"
+			label far
+			`	He pauses for a second, perhaps pondering if he can trust you with the answer.`
+			`	"I'm a holographer from Icelake. I chartered a flight from Icelake to Allhome with my family. They would have gone on vacation while I would have been at a conference. Instead, the human merchants locked us in a cell and took us here."`
+			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them, but they had already landed. The ship was locked down; we could not take off. Since then, we've been hiding near here."`
+			`	"Can you take us back to <destination>? I have considerable savings back there, and can pay you well. Say, <payment>?"`
+			choice
+				`	"Sure!"`
+				`	"No, I can't."`
+					decline
+			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
+				goto leave
+			label "ride offer"
+			`	"Yes! Some crooked human merchants locked us in a cell and took us here. I've been unable to find anyone who has even heard of Hai space, and I'm afraid to approach the Syndicate. You're the first lead I have!"`
+			`	"I promise I'll pay you well when we return. I can afford <payment>... when I'm back at <destination>."`
+			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
+			label leave
+			`	"Follow me back to my house, and we will make our preparations."`
+			choice
+				`	(Follow him.)`
+			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a triad of energetic Hai children eating dinner with their mother.`
+			`	Duniip turns off his hologram and addresses his family, "I found us a pilot! He's going to take us all the way home." This was followed by four cheering family members and one baby's gurgle.`
+			`	The family was already packed to leave. Duniip explains they move regularly so they're not found. It takes them under two minutes to finish preparations and don their holoprojectors.`
+			`	Your walk back to the ship is largely uneventful until some men with strange scanners detect the Yeertle family. One man throws a small grenade with a green light on top. You recognize it as a low-level EMP device. All five holograms turn off, along with all computers within twenty meters. Fortunately, your ship is outside that range.`
+			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
+				launch
+	on complete
```

Comment:
The rest of these missions have log entries upon completion. Was there any particular reason this one didn't?

Reply:
Evidently an oversight on Nomadic's part.